LICENSE-MIT

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015 The spaceapi-server-rs Developers
+Copyright (c) 2014-2015 The spaceapi-server-rs Developers
```

Comment:
I'm probably overthinking this :wink: but here is a relevant quote from [wikipedia](https://en.wikipedia.org/wiki/Copyright_notice#Form_of_Notice):
> The year of first publication. If the work is a derivative work or a compilation incorporating previously published material, **the year date of first publication of the derivative work or compilation is sufficient**. Examples of derivative works are translations or dramatizations; an example of a compilation is an anthology. The year may be omitted when a pictorial, graphic, or sculptural work, with accompanying textual matter, if any, is reproduced in or on greeting cards, postcards, stationery, jewelry, dolls, toys, or useful articles.

So if the year of first publication of the derivative work is sufficient, I assume one can also insert the year of the first publish of the original work.